packages/integration-sdk-testing/src/jest.ts

```diff
@@ -9,15 +9,22 @@ import {
   IntegrationSpecConfig,
   IntegrationStepExecutionContext,
   MappedRelationship,
+  Relationship,
   RelationshipClass,
   Step,
 } from '@jupiterone/integration-sdk-core';
+import { SyncExpectationResult } from 'expect/build/types';
 import { getMatchers } from 'expect/build/jestMatchersObject';
+import { StepTestConfig } from './config';
+import { buildStepDependencyGraph } from '@jupiterone/integration-sdk-runtime';
+import { filterGraphObjects } from './filterGraphObjects';
 
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace jest {
     interface Matchers<R> {
+      toMatchStepMetadata(testConfig: StepTestConfig): R;
```

Comment:
Can these be documented so that when we hover over them in our editors, we see the example use such as you have in the `CHANGELOG.md`?

Reply:
🔥

Reply:
This is fantastic.

Reply:
Yah, nice, thanks so much!